setup.py

```diff
@@ -137,10 +137,28 @@ def get_version(rel_path: str) -> str:
             "sphinx",
             "sphinx_rtd_theme",
             "pre-commit",
+            # CI dependencies
+            "wheel",
+            "xgboost",
+            "catboost",
+            "setuptools",
+            "jupyter",
+            "jupyter_contrib_nbextensions",
+            "black",
+            "pylint",
+            "mypy",
+            "flake8",
+            "readthedocs_sphinx_ext",
+            "cmake",
+            "lxml",
+            "baostock",
+            "yahooquery",
+            "beautifulsoup4",
+            "tianshou",
+            "gym>=0.24",
```

Comment:
gym is already in `REQUIRED`

Reply:
I know about this, but if not, it will cause gym to downgrade, which in turn will cause pytest to fail, a problem I'm also confused about. Would adding a comment to this location be a good solution?

Reply:
Yes,  all these magic conditions need comment.